code-review/README.md

```diff
@@ -65,6 +65,10 @@ experience, refactors the existing code). Then:
 - Seek to understand the author's perspective.
 - Sign off on the pull request with a 👍 or "Ready to merge" comment.
 - Remember that you are here to provide feedback, not to be a gatekeeper.
+- When suggesting changes:
+  - Communicate clearly which lines you suggest adding/removing.
+  - Test the suggested changes to validate it works whenever possible.
+  - When not possible, let the pull request author know that you did not test the suggestion.
```

Comment:
Nice! Maybe also we could say something about providing some context for a change instead of just suggesting it without explaining why it might be better. Would something like this work?

```suggestion
  - When not possible, let the pull request author know that you did not test the suggestion.
  - Provide some context to let the author know why you're suggesting the change.
```

Reply:
Good point. At first I felt like it could be repetitive because I take providing context as a guideline already in this document.

However, I agree over-communicating is better, and maybe someone might land just on this section and not read the other guidelines, or simply not get to the same conclusion as I did 😉  Thank you for your suggestion!